orchagent/portsorch.cpp

```diff
@@ -952,7 +984,11 @@ bool PortsOrch::getPortAdminStatus(sai_object_id_t id, bool &up)
     if (status != SAI_STATUS_SUCCESS)
     {
         SWSS_LOG_ERROR("Failed to get admin status for port pid:%" PRIx64, id);
-        return false;
+        task_process_status handle_status = handleSaiGetStatus(SAI_API_PORT, status);
+        if (handle_status != task_process_status::task_success)
+        {
+            return parseHandleSaiStatusFailure(handle_status);
```

Comment:
>parseHandleSaiStatusFailure(handle_status); [](http://example.com/codeflow?start=19&length=43)

I think you still need to `return false`.

Reply:
Yes, fixed.